### PR TITLE
Update Calico add-on bullet

### PR DESCRIPTION
**Pull request by tmjd**
Adding more detail to the Calico bullet under the Networking and Network Policy addon section.

### CODE REVIEW

**Comment by tmjd** [comment]
@sftim Thanks for the review, PTAL, I pushed up an update to see if that is better.

File: content/en/docs/concepts/cluster-administration/addons.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Add-ons in each section are sorted alphabetically - the ordering does not imply
 
 
 * [ACI](https://www.github.com/noironetworks/aci-containers) provides integrated container networking and network security with Cisco ACI.
-* [Calico](https://docs.projectcalico.org/latest/getting-started/kubernetes/) is a secure L3 networking and network policy provider.
+* [Calico](https://docs.projectcalico.org/latest/introduction/) is a networking and network policy provider. Calico supports a flexible set of networking options so you can choose the most efficient option for your situation, including non-overlay and overlay networks, with or without BGP. Calico uses the same engine to enforce network policy for hosts, pods, and (if using Istio & Envoy) applications at the service mesh layer.
```
Comment (bradtopol) [inline content/en/docs/concepts/cluster-administration/addons.md]: Great description of Calico. I also verified that the new link is working properly.

Comment (bradtopol) [reply content/en/docs/concepts/cluster-administration/addons.md]: Also I believe this now addresses @sftim's concern. @sftim can you verify

**Comment by bradtopol** [comment]
Also I believe the update now addresses @sftim's concern. @sftim can you verify?

File: content/en/docs/concepts/cluster-administration/addons.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/concepts/cluster-administration/addons.md]: Thanks for checking. @bradtopol I'm happy with your opinion, and yes this looks fine to me.

**Comment by sftim** [comment]
/sig network
/approve
/lgtm
@bradtopol hope that's OK?

**Pull request merged**